examples/cassandra/README.md

```diff
@@ -495,39 +506,28 @@ UN  10.244.4.2  32.45 KB   256     100.0%            0b1be71a-6ffb-4895-ac3e-b97
 UN  10.244.3.3  51.28 KB   256     100.0%            dafe3154-1d67-42e1-ac1d-78e7e80dce2b  rack1
 ```
 
-### tl; dr;
-
-For those of you who are impatient, here is the summary of the commands we ran in this tutorial.
-
-```sh
-# create a service to track all cassandra nodes
-kubectl create -f examples/cassandra/cassandra-service.yaml
+**Note**: This example had you delete the cassandra Replication Controller before
+you created the Daemonset.  This is because-- to keep this example simple-- the
```

Comment:
nit: Should this be "This is because --to keep this example simple-- the" so their is a space between because and --, or is the current format intentional?